frontend/components/about/ValuesItems.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+
+    import type { values } from '.';
+    const props = defineProps<values>();
```

Comment:
Линтер пишет, что переменная `props` нигде не используется
```suggestion
    defineProps<values>();
```